vm-virtio/src/device.rs

```diff
@@ -13,6 +13,9 @@ use std::sync::Arc;
 use vm_memory::GuestMemoryMmap;
 use vmm_sys_util::EventFd;
 
+pub type InterruptClosure =
```

Comment:
Does this have to be a public type?
If it does, should it be `VirtioInterruptClosure` instead?

Reply:
I think it has to be public since the other `vm-virtio` rust modules (`block`, `net`, `rng` and `transport::pci_device`) need it. WDYT?

About the naming, I like `VirtioInterruptClosure`, I'll make the change.